userguide/plugins.rst

```diff
@@ -248,6 +248,84 @@ are permanently deleted.
+   | :file:`post_install.sh` | This script is run *inside* the jail after                           |
+   |                         | it has been created and packages installed.                          |
+   |                         | Enable services in /etc/rc.conf that need to start                   |
+   |                         | with the jail startup, apply configuration settings,                 |
+   |                         | and more.                                                            |
+   +-------------------------+----------------------------------------------------------------------+
+   | :file:`ui.json`         | Json file that accepts the  key or value options:                    |
+   |                         |                                                                      |
+   |                         | :samp:`adminportal: "http://%%IP%%/"`                                |
+   |                         |                                                                      |
+   |                         | The web-interface of the plugin for control and                      |
+   |                         | configuration .                                                      |
+   +-------------------------+----------------------------------------------------------------------+
+   | :file:`overlay/`        | Directory of files copied on top of the jail post-install.           |
+   |                         | For example, :file:`usr/local/bin/myfile` is placed in the jails     |
+   |                         | /usr/local/bin/myfile location. Can be used to supply custom         |
+   |                         | files and configuration data, scripts, and more.                     |
+   +-------------------------+----------------------------------------------------------------------+
+   | :file:`settings.json`   | Json file that controls plugins settings interface. The required     |
+   |                         | fields include:                                                      |
+   |                         |                                                                      |
+   |                         | * :samp:`"servicerestart" : "service plexmediaserver restart"`       |
+   |                         |                                                                      |
+   |                         | Command to run when restarting service after changing settings.      |
+   |                         |                                                                      |
+   |                         | * :samp:`"serviceget" : "/usr/local/bin/myget"`                      |
+   |                         |                                                                      |
+   |                         | Points to the command used to get values for plugin configuration.   |
+   |                         | Provided by the plugin creator. Can be any language as long as it    |
```

Comment:
Does "any language" mean "any programming language" or "any human language"?  Not clear what "accepts" means.  Is this a runnable command?